app/views/projects/classifies/_sidebar.html.erb

```diff
@@ -7,6 +7,8 @@
                          project: project,
                          state_transitions: state_transitions } %>
 
+    <%= link_to _('Skip'), project_classify_path(project, url_title: info_request.url_title), method: :patch, class: 'button-tertiary button--full-width form__skip-button' %>
```

Comment:
oops need to trim the line length of this and remove the JS-based button (and the skip JS while I'm here)